order.js

```diff
@@ -33,16 +33,120 @@ Render.main = (data) => {
 
     <h3 id="proofs">Proofs</h3>
     ${Render.try('proofs', data)}
+
+    <h3 id="messages">Message History</h3>
+    <div class="section">
+      ${Render.try('messages', data)}
```

Comment:
Just to explain, one might ask why not simply write
`${Render.messages(data)}`
The reason it's wrapped in try, is it allows each section called by Try to fail gracefully and report to mdb about that failure.
So if something is wrong with the "messages" render function the rest of page will still render.